tests/mock_network/accumulate.rs

```diff
@@ -57,10 +58,14 @@ fn messages_accumulate_with_quorum() {
     expect_no_event!(nodes[0]);
 
     // If there are `quorum` senders but they all only sent hashes, nothing can accumulate.
-    // Only after `nodes[0]`, which is closest to `src.name()`, has sent the full message, it
-    // accumulates.
+    // Only after one of the first three nodes, which are closest to `src.name()`, has sent the full
+    // message, it accumulates.
     let message_id = MessageId::new();
-    for node in nodes.iter_mut().skip(1).take(quorum) {
+    for node in nodes
+        .iter_mut()
+        .skip(3)
```

Comment:
not a fan of hardcoding this 3. Maybe: `let delivery_group_size = min_section_size + 2 / 3;`?
Maybe we can even make this `+ 2 / 3` a function because I'm pretty sure it's logic I've seen duplicated in a few places.